docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -1,17 +1,37 @@
-# Migrating to v6
+# Upgrade to v6
```

Comment:
We should probably update the corresponding sidebar menu item, and probably the url, too (with a redirect to not break existing links to this page). Maybe in another PR.

<img width="259" alt="image" src="https://github.com/user-attachments/assets/11166c41-ed08-4179-9f0e-316cb5a3ec0b">

Reply:
Done